ts.eclipse.ide.angular2.cli/src/ts/eclipse/ide/angular2/internal/cli/launch/AngularCLILaunchConfigurationDelegate.java

```diff
@@ -61,24 +61,30 @@ public void launch(ILaunchConfiguration configuration, String arg1, ILaunch laun
 			return;
 		}
 
+		IPath processWorkingDir = workingDir;

```

Comment:
@Springrbua it was my last code that I have removed, please synchronize with last source, thanks!